grid/example/src/cells/RowEditOverlay.js

```diff
@@ -0,0 +1,215 @@
+    marginLeft: "1244px"
+}
+const RowEditOverLay = memo((props) => {
+    const { row, airportCodeList, updateCellData } = props;
+    const [value, setValue] = useState(row.original.flight);
+    const [cellSegmentValue, setCellSegmentValue] = useState(row.original.segment);
+    const [cellWeightValue, setCellWeightValue] = useState(row.original.weight);
+    const [srValue, setSrValue] = useState(row.original.sr);
+    const [remarksValue, setRemarksValue] = useState(row.original.remarks);
+
+
+    const onChangeSaveRemarks=(e)=>{
+    setRemarksValue(e.target.value)
+    }
+    const onChangeSaveSr = (e) => {
+        setSrValue(e.target.value);
+    };
+
+
+    const onWeightPercentageChange = (e) => {
+        setCellWeightValue({
+            ...cellWeightValue,
+            percentage: e.target.value
+        });
+    };
+
+    const onWeightValueChange = (e) => {
+        setCellWeightValue({
+            ...cellWeightValue,
+            value: e.target.value
+        });
+    };
+
+    const onChangeFrom = (e) => {
+        setCellSegmentValue({
+            ...cellSegmentValue,
+            from: e.target.value
+        });
+    };
+
+    const onChangeTo = (e) => {
+        setCellSegmentValue({
+            ...cellSegmentValue,
+            to: e.target.value
+        });
+    };
+
+    const onChangeSaveFlightNo = (e) => {
+        setValue({
+            ...value,
+            flightno: e.target.value
+        });
+    }
+
+    const onDateChange = (e) => {
+        setValue({
+            ...value,
+            date: getValueOfDate(e.target.value)
+        });
+    };
+
+    /**
+     * Method to Convert Date to required Format as per value of type
+     * @param {String} inputDate 
+     * @param {String} type 
+     */
+    const getValueOfDate = (dateValue, type) => {
+        const date = new Date(dateValue);
+        if (type === "calendar") {
+            const dateTimeFormat = new Intl.DateTimeFormat("en-US", { year: "numeric", month: "2-digit", day: "2-digit" });
+            const [{ value: month }, , { value: day }, , { value: year }] = dateTimeFormat.formatToParts(date);
+            return `${year}-${month}-${day}`;
+        } else {
+            const dateTimeFormat = new Intl.DateTimeFormat("en-US", { year: "numeric", month: "short", day: "2-digit" });
+            const [{ value: month }, , { value: day }, , { value: year }] = dateTimeFormat.formatToParts(date);
+            return `${day}-${month}-${year}`;
+        }
+    };
+
+    const saveChangesForRowEdit = (row) => {
+        updateCellData(row.index, "flight", value);
+        updateCellData(row.index, "segment", cellSegmentValue);
+        updateCellData(row.index, "weight", cellWeightValue);
+        updateCellData(row.index, "sr", srValue);
+        updateCellData(row.index, "remarks", remarksValue);
+    }
+    return (
+        <div className="main-div" style={divStyle}>
+            {/* <ClickAwayListener onClickAway={closeOverlay}> */}
```

Comment:
Lets avoid keeping commented codes in the main repo

Reply:
Removed